app/helpers/policies_helper.rb

```diff
@@ -20,7 +20,7 @@ def deploy_by_radios(f, policy)
       end
 
       help_block = content_tag(:span, :class => 'help-block help-inline') do |variable|
-        tool.help_text unless tool.available?
+        config_inline_help tool.inline_help unless tool.available?
```

Comment:
This does not show anything, the condition should probably be using `if`. I think we could go further with the description and also display it more standard way using label help. Please see https://github.com/xprazak2/foreman_openscap/pull/1 and merge if you agree

Reply:
Seems like we have different ideas about the help. It showed when `tool.available? #=> false`, and it told users what to do to enable it.